spec/features/dams_object_spec.rb

```diff
@@ -1717,3 +1719,47 @@
     expect(page).not_to have_content("Update DOI record")
   end
 end
+
+describe "vrr user who has authorized works should not see Download File and Embed button" do
+  let!(:user) { create_auth_link_user }
+
+  before(:all) do
+    @unit = DamsUnit.create pid: 'xx48484848', name: "Test Unit", description: "Test Description",
```

Comment:
I think this also needs to be deleted in the `after(:all)` block?

Reply:
Updated, thanks for catching it.